src/dosbox.cpp

```diff
@@ -821,6 +821,12 @@ void DOSBOX_Init()
 	        "                300 200 (Wait 300ms before fading out over a 200ms period)\n"
 	        "                1000 3000 (Wait 1s before fading out over a 3s period)");
 
+	pbool = secprop->Add_bool("opl_remove_dc_bias", when_idle, false);
+	pbool->Set_help(
+	        "Remove DC bias from the OPL output. This should only be used as a last resort\n"
+	        "to fix popping in games that play PCM using the Adlib card, such as is the case\n"
```

Comment:
Minor clarification suggestion: "that play PCM audio using the OPL synthesiser on a Sound Blaster or AdLib card"

Reply:
added